oidc-client-app-samples/ios-client-app-sample/WSO2-IS-SampleAppTests/WSO2_IS_SampleAppTests.swift

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
Add the WSO2 Licence header.

Reply:
Fixed in https://github.com/wso2/samples-is/pull/168